test/RelayClient.test.ts

```diff
@@ -549,71 +587,30 @@ contract('RelayClient', function (accounts) {
     } catch (ignored) {
       assert.isTrue(ignored.otherErrors.length > 0,
         'There were no lookup errors, this is not the exception we are aiming for')
+      // @ts-ignore
       assert.isTrue(rc.failedRelays[relayUrl] !== undefined)
     }
   })
 
-  describe('relay balance management', async function () {
```

Comment:
where all `relay balance management` tests moved?

Reply:
This is a 'relay client' test file, but the test is checking behavior between owner and server that does not currently exist. Either I or Shahaf will add these tests to the server test suite where they belong.

Reply:
then skip and mark why - but don't delete.